PolyEngine/Core/Src/Path.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "Defines.hpp"
+#include "Dynarray.hpp"
+#include "String.hpp"
+
+namespace Poly {
+
+	class CORE_DLLEXPORT Path
+	{
+	public:
+		static Path* Init();
+		//TODO when String.split() will be ready
+		static Path* FromString(const String& source);
+
+		Path& RootAt(const String& root);
```

Comment:
There should also be a way to convert relative path to absolute. This will make possible using relative paths to create it and then passing absolute path somwhere else.